packages/asc-ui/src/components/Body/Body.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
```

Comment:
Don't think this should be in here, as this is very specific. Have a look at the `atlas` repository where a container component specifically for that project is used

Reply:
I agree. Let's have a look what might be missing in the grid as well perhaps?

Reply:
We need to soort this out. The current version of the Header is also too specific so we need a Body component if we want to cover all the Header state scenario's. Perhaps a basic header and an atlas specific Header+Body component will adress all the needs.